editor/src/clj/editor/sprite.clj

```diff
@@ -55,7 +57,7 @@
 ; Render assets
 (vtx/defvertex texture-vtx
   (vec4 position)
-  (vec2 texcoord0 true)
+  (vec2 texcoord0)
```

Comment:
The `true` here tagged the attribute for normalization in the shader. Such an attribute would result in a [-1.0 1.0] range in the shader if encoded as an integer type. But since this is a float value, it is meaningless.